sql/hive/src/main/scala/org/apache/spark/sql/hive/client/HiveClientImpl.scala

```diff
@@ -229,8 +230,8 @@ private[hive] class HiveClientImpl(
         case e: Exception if causedByThrift(e) =>
           caughtException = e
           logWarning(
-            "HiveClient got thrift exception, destroying client and retrying " +
-              s"(${retryLimit - numTries} tries remaining)", e)
+            log"HiveClient got thrift exception, destroying client and retrying " +
+              log"${MDC(RETRY_COUNT, numTries)} times", e)
```

Comment:
Although there is a `slight difference` in the meaning expressed in these two places, `one` is how many times are `left` and the `other` is how many times have been `retried`. In order to be consistent with `other parts of Spark`, adjustments have been made here